src/main/resources/assets/itemzoom/lang/en_us.lang

```diff
@@ -1,21 +1,21 @@
 # Key Bindings
-key.itemzoom.toggle=Toggle
-key.itemzoom.hold=Hold to Enable
+key.itemzoom.toggle=Toggle Item Zoom
+key.itemzoom.hold=Hold To Display
 key.itemzoom.zoom.in=Zoom In
 key.itemzoom.zoom.out=Zoom Out
 
 # Config
-config.itemzoom.toggle.enabled=Toggle Enabled
-config.itemzoom.toggle.enabled.comment=True if Item Zoom has been toggled On.
+config.itemzoom.title=%MODNAME Config
+
+config.itemzoom.toggle.enabled=Toggle Item Zoom
+config.itemzoom.toggle.enabled.comment=If set to "false", Item Zoom will be disabled.
 config.itemzoom.zoom.amount=Zoom Amount
-config.itemzoom.zoom.amount.comment=Set lower to make the item zoom less.
-config.itemzoom.jei.only=JEI List Only
-config.itemzoom.jei.only.comment=Only zoom items from the JEI list.
+config.itemzoom.zoom.amount.comment=Set lower amount to make the item zoom less.
+config.itemzoom.jei.only=JEI Overlay Only
+config.itemzoom.jei.only.comment=Zoom items only from the JEI overlay with ingredients list.
 config.itemzoom.show.help.text=Show Help Text
```

Comment:
Didn't you think "display" would be better than "show" here and below? "Show" suggests that e.g. this text is displayed but it's hidden (like with 0% opacity). I don't know how this feature is written in Java, but I think it displays/doesn't display objects rather than shows/hides them.

Reply:
I agree "Display" is better. 
Often when writing these, I am in a coding mindset and find it difficult to find the right words from a player perspective.

Reply:
Yeah, I agree with you. It's even harder when English is not your native language (which applies e.g. to me). Sometimes I have similar problems during translation. Anyway, I'm going to change the words "show" to "display".

You could later change "config.itemzoom.show.\*" to "config.itemzoom.display.\*" to prevent them from being confusing in the future, but it's just a suggestion.

Reply:
Yeah, I agree with you. It's even harder when English is not your native language (which applies e.g. to me). Anyway, I'm going to change the words "show" to "display".

You could later change "config.itemzoom.show.\*" to "config.itemzoom.display.\*" to prevent them from being confusing in the future, but it's just a suggestion.